### PR TITLE
Fix code formatting in Detailed Shared Raw Disks feature page

### PR DESCRIPTION
**Pull request by rollandf**
Changes proposed in this pull request:

- Fix code formatting
- Fix URLs

I confirm that this pull request was submitted according to the [contribution guidelines](/.github/CONTRIBUTING.md): @rollandf

This pull request needs review by: @mureinik @danielerez @maorlipchuk

### CODE REVIEW

File: source/develop/release-management/features/storage/detailedsharedrawdisk.html.md [hunk 1]
```diff
@@ -175,14 +175,8 @@ Attaching shareable disk will not consume new Quota resource. Affected oVirt pro
 
 ### Documentation / External references
 
-<http://www.ovirt.org/wiki/Features/SharedRawDisk>
-
-### Comments and Discussion
-
-<http://www.ovirt.org/wiki/Talk:Features/SharedRAWDisk>
+[Features/SharedRawDisk](Features/SharedRawDisk)
```
Comment (mureinik) [inline source/develop/release-management/features/storage/detailedsharedrawdisk.html.md]: Just remove this entire section. No sense in a page linking to itself.

File: source/develop/release-management/features/storage/detailedsharedrawdisk.html.md [hunk 2]
```diff
@@ -175,14 +175,8 @@ Attaching shareable disk will not consume new Quota resource. Affected oVirt pro
 
 ### Documentation / External references
 
-<http://www.ovirt.org/wiki/Features/SharedRawDisk>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/detailedsharedrawdisk.html.md]: This leaves a bare header - remove it too please

File: source/develop/release-management/features/storage/detailedsharedrawdisk.html.md [hunk 3]
```diff
@@ -175,14 +175,8 @@ Attaching shareable disk will not consume new Quota resource. Affected oVirt pro
...
-
-### Comments and Discussion
-
-<http://www.ovirt.org/wiki/Talk:Features/SharedRAWDisk>
+[Features/SharedRawDisk](Features/SharedRawDisk)
 
 ### Future Work
 
 ### Open Issues
```
Comment (mureinik) [inline source/develop/release-management/features/storage/detailedsharedrawdisk.html.md]: Here too

**Comment by rollandf** [comment]
It seems there are two pages.
The one I worked on:
http://www.ovirt.org/develop/release-management/features/storage/detailedsharedrawdisk/
and:
http://www.ovirt.org/develop/release-management/features/storage/sharedrawdisk/

It seems redundant, I will merge the two and remove the second one.

**Comment by mureinik** [comment]
@rollandf This is how we used to do things in the old MediaWiki, and indeed doesn't fit the new site. You should remove http://www.ovirt.org/develop/release-management/features/storage/sharedrawdisk/ and **rename**  http://www.ovirt.org/develop/release-management/features/storage/detailedsharedrawdisk/ to it, as that is the sensible name. Additionally, you should edit the redirects file in case someone has an onld link. See, e.g., what I did with the SEAT pages: https://github.com/oVirt/ovirt-site/commit/b62d9cd6bd261fbc38e5b8c456e8dcfbe52fffec

**Comment by mureinik** [comment]
@rollandf Can you please rebase this PR so it can be merged?

**Comment by rollandf** [comment]
Rebased

**Pull request merged**